### PR TITLE
Use completion kind as detail if no detail provided

### PR DESCRIPTION
**Pull request by tomv564**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1614,6 +1638,10 @@ def format_completion(self, item) -> 'Tuple[str, str]':
         # Sublime handles snippets automatically, so we don't have to care about insertTextFormat.
         label = item.get("label")
         detail = item.get("detail")
```
Comment (rwols) [inline main.py]: Should have a default of `None` here, then?

File: main.py [hunk 2]
```diff
@@ -95,6 +95,30 @@ class SymbolKind(object):
...
+    Value = 12
+    Enum = 13
+    Keyword = 14
+    Snippet = 15
+    Color = 16
+    File = 17
+    Reference = 18
+
+
+completion_item_kind_names = {v: k for k, v in CompletionItemKind.__dict__.items()}
```
Comment (rwols) [inline main.py]: Why not just write it as a dict in the first place?

Comment (randy3k) [reply main.py]: or [enum](https://docs.python.org/3.4/library/enum.html).

Comment (tomv564) [reply main.py]: Had wondered about Enum before, but documentation said "new in version 3.4".
Tried it now, getting an ImportError:
```
  File "/Users/tomv/Library/Application Support/Sublime Text 3/Packages/LSP/main.py", line 7, in <module>
    from enum import Enum
ImportError: No module named 'enum'
```

Comment (randy3k) [reply main.py]: Ai...forget that 3.3 doesn't support enum.

Comment (tomv564) [reply main.py]: It was easiest to paste and fix the list from the spec or typescript and then add the one-liner to create the reverse lookup. 
I agree the lookup is the only thing we use right now, but I'd like an enum-like lookup for future functionality.

Comment (rwols) [reply main.py]: Fair enough 👍

**Pull request merged**